docs/standard/containerized-lifecycle-architecture/run-manage-monitor-docker-environments/index.md

```diff
@@ -12,7 +12,9 @@ Vision: Enterprise applications need to run with high availability and high scal
 
 This last pillar in the containerized Docker applications life cycle is centered on how you can run, manage, and monitor your applications in scalable, high availability (HA) production environments.
 
-How you run your containerized applications in production (infrastructure architecture and platform technologies) is also very much related and completely founded on the chosen architecture and development platforms that we looked at in the Chapter 1 of this e-book. This chapter examines specific products and technologies from Microsoft and other vendors that you can use to effectively run highly scalable, HA distributed applications plus how you can manage and monitor them from the IT perspective.
+The way you run your containerized applications in production (infrastructure architecture and platform technologies) is  very much related and founded on the chosen architecture and development platforms we looked at in Chapter 1 of this e-book.
```

Comment:
```suggestion
The way you run your containerized applications in production (infrastructure architecture and platform technologies) is very much related and based on the chosen architecture and development platforms discussed in Chapter 1 of this e-book.
```

Reply:
Yep, sorry, sometimes spanglish creeps in 😅